plugins/parsers/registry.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/influxdata/telegraf/plugins/parsers/graphite"
 	"github.com/influxdata/telegraf/plugins/parsers/grok"
 	"github.com/influxdata/telegraf/plugins/parsers/influx"
+	"github.com/influxdata/telegraf/plugins/parsers/influx_upstream"
```

Comment:
Do you think it would be better to move influx_upstream to be a subdirectory of plugins/parsers/influx? That way people wouldn't see it and assume it's a standalone parser they might want to use. Then we also wouldn't need a influx_upstream/README.md

Reply:
agreed and done